tests/roles/development_environment/tasks/main.yaml

```diff
@@ -46,3 +46,18 @@
 - name: debug token
   ansible.builtin.debug:
     var: before_adoption_token
+
+- name: creates credential and stores it
+  no_log: "{{ use_no_log }}"
+  environment:
+    INSTALL_YAMLS_PATH: "{{ install_yamls_path }}"
+    EDPM_PRIVATEKEY_PATH: "{{ edpm_privatekey_path }}"
+    OS_CLOUD_IP: "{{ standalone_ip | default(edpm_node_ip) }}"
+    OS_CLOUD_NAME: "{{ os_cloud_name }}"
+    OS_USER: "{{ os_user }}"
```

Comment:
The user name has been changed to SOURCE_OSP_SSH_USER: "{{ source_osp_ssh_user }}"